data/json/monsterdrops/zombie_default.json

```diff
@@ -26,7 +27,13 @@
                     "collection": [
                       { "group": "pants_male", "prob": 60, "damage": [ 1, 4 ] },
                       { "group": "shirts_unisex", "prob": 60, "damage": [ 1, 4 ] },
-                      { "item": "leather_belt", "prob": 30, "damage": [ 1, 4 ] }
+                      {
+                        "distribution": [
+                          { "item": "leather_belt", "prob": 10, "damage": [ 1, 4 ] },
+                          { "group": "christmas_belts", "prob": 90, "damage": [ 1, 4 ], "event": "christmas" }
```

Comment:
Even during christmas 90% of the zombies shouldn't be wearing christmas items.   I would think.